src/main/java/com/powsybl/openloadflow/network/PiModel.java

```diff
@@ -50,12 +50,17 @@ public interface PiModel {
     void roundR1ToClosestTap();
 
     enum Direction {
+        NONE,
         INCREASE,
-        DECREASE
+        DECREASE,
+        INCREASE_THEN_DECREASE,
```

Comment:
We should not have INCREASE_THEN_DECREASE and DECREASE_THEN_INCREASE in the enum because it is also the one used for phase shifter with meaningless values in this case. Also the memory of past direction should be stored in another way: for instance with a counter a direction change.